packages/tiered/src/index.d.ts

```diff
@@ -0,0 +1,23 @@
+import {EventEmitter} from 'events';
+import Store from 'keyv';
+
+export = KeyvTiered;
+declare class KeyvTiered extends EventEmitter {
+	constructor(options: KeyvTiered.Options);
+	set(key: string, value: string | Record<string, unknown> | undefined): Promise<any>;
+	get(key: string): Promise<string | undefined>;
+	getMany(keys: string[]): Promise<string[] | undefined>;
+	delete(key: string): boolean;
+	deleteMany(keys: string[]): boolean;
+	clear(): Promise<void>;
+	// Iterator(namespace: string | undefined): AsyncGenerator<any, void, any>;
```

Comment:
Is Iterator not valid for this?

Reply:
I haven't tried with an iterator. I will try and see